ACKategoriesCore/RandomExtensions/Date+Random.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+public extension Date {
+    /// - Parameters:
+    ///     - min: Minimum `timeIntervalSince1970`
+    ///     - max: Maxium `timeIntervalSince1970`
+    /// - Returns: Random date
+    static func random(min: Int = 0, max: Int = 2000000000) -> Date {
```

Comment:
Why `2000000000`? It's just a detail but maybe we could use `Int.max`?

Reply:
Yeah, I agree that would make sense 👍